src/_P076_HLW8012.ino

```diff
@@ -428,42 +438,42 @@ boolean Plugin_076(uint8_t function, struct EventStruct *event, String& string)
           float HLW[7];
           p076_hpowerActive = Plugin_076_hlw->getActivePower(valid);
 
-          if (valid) {
+          if (valid || P076_TOZERO) {
```

Comment:
This just as an example, but shouldn't the code be like this:

```c++
if (valid || P076_TOZERO) {
  HLW[2] = valid ? p076_hpowerActive : 0;
  success = true;
}
```

Reply:
The hlw8012 lib is returning a zero with valid=false.

Reply:
For all of those functions?

Reply:
getVoltage, getCurrent and getActivePower are returning 0 with valid=false if there are no measured pulses.

getApparentPower returning also 0 because voltage (0) * current (0) is 0.

getReactivePower and getPowerFactor also returning 0 as the values which are used to calculate are 0.

Reply:
OK, so let's hope there is no divide-by-zero there then :) (would have been a previous bug if it were)